Alicerce.xcodeproj/project.pbxproj

```diff
@@ -1012,6 +1040,23 @@
 		};
 /* End PBXResourcesBuildPhase section */
 
+/* Begin PBXShellScriptBuildPhase section */
+		F12FB1BF20A6E8CD00AA3A52 /* [SwiftLint] Lint files */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputPaths = (
+			);
+			name = "[SwiftLint] Lint files";
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "if which mint >/dev/null; then\n    mint run realm/SwiftLint@0.22.0 swiftlint\nfi\n";
```

Comment:
I need to test installing Alicerce using carthage. But I was under the impression that Carthage builds the target, and if it wouldn't find mint installed it would fail the build for people using Carthage, which I think is not desirable

Reply:
Making that change and trying out Carthage on a test project I got an error. Without this, it worked without an issue. Therefore I think we should not force Mint to be installed.

Reply:
Ok, didn't think about that, makes sense 😅

Even though I would prefer for it to fail, I understand that because of Carthage we can't force it to be installed. 

Its just that any contributor without it installed will not be "aware" that Mint's missing until it breaks after opening up a PR (assuming the changes would cause a linter issue).

Possibly a nice middle ground would be to fire a warning if Mint was missing, but don't know if it's easy to do that from a shell script.

Reply:
I will look into throwing a warning about Mint not being installed.

Reply:
I think that is not possible to throw an warning on the from a shell script, they are basically based on exit status codes (simple bash).
Perhaps send an echo with a warning? i know, it's not ideal and perhaps most of the developers would miss that, but, just my 2 cents.

`echo "⚠️⚠️☢️ ... 😂"`

Reply:
It's possible as I used something similar before. You can check an example [here](http://jeffreysambells.com/2013/01/31/generate-xcode-warnings-from-todo-comments).